architecture-decision-records/0033-dta-review-user-permissions.md

```diff
@@ -0,0 +1,30 @@
+## 33. User permissions for DTA Review feature
+
+## Context
+
+Work is underway for the TDR web application to support a new step in the standard transfer journey whereby a TNA Digital Transfer Advisor can review consignment metadata input by a transferring body and either approve (allowing them to proceed with transfer) or reject (requiring metadata changes before the transfer can proceed).
+
+The Digital Transfer Advisor will be able to download metadata associated with a consignment for which a review has been requested in spreadsheet form. When they submit a review decision, it will require an update in the table recording the state of the consignment they are reviewing.
+
+This will require a departure from our existing user permission model, where calls to fetch or amend data in the consignment API can only be successful when using the authorisation token of the user who originally uploaded the data.
```

Comment:
Wording change: `... departure from TDR's existing user ...`

I would maybe add an additional sentence to this paragraph: `TDR's original permissions design was based on the requirement that only a user who uploaded the consignment would need access to the consignment.`